src/dialogs/more-info-dialog.html

```diff
@@ -16,6 +16,11 @@
     <style>
       paper-dialog {
         font-size: 14px;
+        width: 450px;
+      }
+
+      paper-dialog[data-domain=camera] {
+        min-width: 450px;
```

Comment:
For camera, we should set width to auto again or else we will show smaller than we can camera images